File: io.py
import numpy as np

A = np.random.rand(2, 2).astype(float)
print(A)

with open("abc", "wb") as f:
    f.write(A)

with open("abc", mode = "rb") as f:
    A = np.frombuffer(f.read(), dtype = float).reshape(2, 2) 
    # A = A.T.reshape(2, 2) if file matrix was column major
    print(A)

File: io.jl
A =
[
    13 11 12 22
    -1 -1 -3 -4
    -4 -2 -2 -9
    -8 -2  0 -5
     0 -6 -7 -4
]

open(io -> write(io, A), "abc", "w")

B = Matrix{Int64}(undef, (5, 4))
open(io -> read!(io, B), "abc", "r")
# B = reshape(B', 5, 4) if file matrix was row major

rm("abc")
display(B)